src/lib/components/SelectField/_components/OptionField.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const OptionField = ({
+  id,
+  option,
+}) => (
+  <option
+    disabled={option.disabled}
+    id={id && `${id}__item__${option.value}`}
```

Comment:
Typically, we compose IDs only within one component. It means that this component should only accept `id` and `${id}__item__${option.value}` composition should be made in `SelectField` (also because of fact that it must be documented in props, so it must be in sync). This is the way how it is done in another components (see Modal for example).